src/pages/single video/notes/modal/EditNotesModal.jsx

```diff
@@ -0,0 +1,69 @@
+import {useState} from "react";
+import { useNotes } from "../../../../context";
+import classes from './notesModal.module.css';
+const EditNotesModal = ({note,setShowEditForm,setPlaying}) => {
+    const {updateNote} = useNotes();
+    const [editedNote,setEditedNote]=useState({
+      ...note
+    })
+
+
+    const changeHandler=(e)=>{
+        const {name,value}=e.target;
+        setEditedNote({
+            ...editedNote,
+            [name]:value
+        })
+    }
+    const submitHandler=async (e)=>{
+        try{
+            e.preventDefault();
+            setShowEditForm();
+            await updateNote(editedNote);
+            setPlaying();
+        }catch(error){
+            console.log(error);
+        }
+    }
+  return (
+    <div className={classes["notes-modal"]}>
+      <form
+        action="post"
+        className={classes["notes-form"]}
+        onSubmit={submitHandler}
+      >
+        <i
+          onClick={setShowEditForm}
+          className={`fas fa-times text-white ${classes["close-btn"]}`}
+        ></i>
+        <h3 className="text-white">Edit note</h3>
+        <label htmlFor="title">
+          Title <br />
+          <input
```

Comment:
you can make either the label or input a block element and avoid the `<br />` tag

Reply:
sure i'll avoid br tags in final refactor